Setup/toc.yml

```diff
@@ -1,5 +1,2 @@
 - name: Setup
-  href: index.md
-  items:
-    - name: Starting our first small NMF project
-      href: FirstNmfProject.md
\ No newline at end of file
+  href: index.md
```

Comment:
Ich glaube, ich würde Setup wieder komplett rausnehmen, wenn da sonst keine weiteren Inhalte drin stehen

Reply:
Ich würde das drinne lassen, einmal wegen der .net sdk version da sonst nirgendwo im Projekt steht welche Versionen kompatibel sind, dazu kommt noch der edge case mit der Package Manager Console. Die PM gibts zum beispiel auf keiner vs version in Mac.